fix_up/models.py

```diff
@@ -38,9 +38,10 @@ class ContractorProject(models.Model):
     contractor = models.ForeignKey('Contractor', on_delete=models.PROTECT)
     contractor_choice = models.IntegerField()
     user_choice = models.BooleanField()
+    seen = models.BooleanField()
     completed = models.BooleanField()
     contractor_before_picture = models.CharField(null=False, max_length=1000)
-    contractor_after_picture = models.CharField(null=False, max_length=1000)
+    contractor_after_picture = models.CharField(default=False, null=False, max_length=1000)
```

Comment:
I'm not sure why we would want to default a char field to false, null might work better here

Reply:
Good catch - I'll change it to `None`